classes/class-wc-connect-functions.php

```diff
@@ -43,16 +43,16 @@ public static function get_wc_helper_auth_info() {
 		}
 
 		/**
-		 * Check if we are currently in Rest API request for the wc/store/cart API call.
+		 * Check if we are currently in Rest API request for the wc/store/cart or wc/store/checkout API call.
 		 *
 		 * @return bool
 		 */
 		public static function is_store_api_call() {
 			if ( ! WC()->is_rest_api_request() && empty( $GLOBALS['wp']->query_vars['rest_route'] ) ) {
 				return false;
 			}
-
-			return false !== strpos( $GLOBALS['wp']->query_vars['rest_route'], 'wc/store/cart' );
+			$rest_route = $GLOBALS['wp']->query_vars['rest_route'];
+			return ( false !== strpos( $rest_route, 'wc/store/cart' ) ) || ( false !== strpos( $rest_route, 'wc/store/checkout' ) );
```

Comment:
How about:

```suggestion
			return 1 === preg_match( '#wc/store/(cart|checkout)#', $rest_route );
```

or:

```suggestion
			return (
				false !== strpos( $rest_route, 'wc/store/cart' ) ||
				false !== strpos( $rest_route, 'wc/store/checkout' )
			);
```

In the latter, aside from the change in indentation, the brackets surrounding individual conditions were also removed.

Reply:
Cool, merging the later.